openadapt/app/util.py

```diff
@@ -50,9 +86,21 @@ def on_export(dest):
     ui.notify("Exported data.")
 
 
-def sync_switch(switch, prop):
+def sync_switch(switch: Any, prop: Any) -> None:
```

Comment:
```
switch: nicegui.elements.switch.Switch,
```

@0dm can you please clarify what is prop here? This function only appears to be used once, and the expected value of `prop` is not clear.

Reply:
this is from `elements.mixins.value_element.ValueElement` (ValueElement)
`from nicegui import ui, elements`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/80a53e0c92855a46739d37d6fe0c5cadbdef1bd0